packages/common/src/FormattingUtils.js

```diff
@@ -117,6 +119,30 @@ function addSign(number) {
   }
 }
 
+// Take an amount based on fromDecimals and convert it to an amount based on toDecimals
+// For example amount = 100 usdt, its decimals are 6. You want to convert it to 18.
+// convertDecimals(6,18)(100000000)  => 1000000000000000000000.
+// Returns a BigNumber you will need to call toString on
+// fromDecimals: number - decimal value of amount
+// toDecimals: number - decimal value to convert to
+// bn: toBN function - optionally provide your own BN function.
+// return => (amount:string)=>BN
+const ConvertDecimals = (fromDecimals, toDecimals, bn = toBN) => {
```

Comment:
OOC why is this different than parseFixed and formatFixed

Reply:
this was needed for a call. converting from synthetic decimals to collateral decimals in the emp.